spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module FeatureHelpers
+  def start_new_application
+    visit '/'
+    click_link 'Start now'
+  end
+end
+
+RSpec.configure do |c|
+  c.include FeatureHelpers, type: :feature
+end
```

Comment:
`applications_new_path` isn't a thing any more. It used to be a GET route, which created a new application and redirected to the build. I've made the initial button a POST, which routes to `/applications` and that creates + redirects. 

That means we're not creating objects in a GET action any more, which is correct from RESTful perspective. But we therefore can't simply visit a page to create a new `Application`.